modules/hardware.nix

```diff
@@ -67,6 +67,7 @@ in {
       };
       loadAddress = mkOption { type = types.ints.unsigned; default = null; };
       entryPoint = mkOption { type = types.ints.unsigned; };
+      alignment = mkOption { type = types.ints.unsigned; default = null; };
```

Comment:
can you put a description on this to say what it's the alignment of?

Reply:
Yep.

Reply:
Done.